src/draive/gemini/lmm_generation.py

```
@@ -127,142 +155,358 @@ async def lmm_completion(  # noqa: C901, PLR0912, PLR0915
+            )
+            ctx.record(
+                ObservabilityLevel.INFO,
+                event="lmm.tool_requests",
+                attributes={"lmm.tools": [call.tool for call in tool_requests]},
+            )
+            return completion_tool_calls
+        elif lmm_completion:
+            ctx.record(ObservabilityLevel.INFO, event="lmm.completion")
+            return lmm_completion
+        else:
+            raise GeminiException("Invalid Gemini completion, missing content!")
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)